linux/Jenkinsfile

```diff
@@ -51,29 +51,29 @@ pipeline {
                         userRemoteConfigs: [[url: "$gitrepo"]]
                     ]
                 )
-                dir("linux") {
-                    stash name: "installercode", includes: "**"
+                dir('linux') {
+                    stash name: 'installercode', includes: '**'
                 }
             }
         }
-        stage ("BUILD")
+        stage('BUILD')
         {
-            parallel{
+            parallel {
                 stage('Build Installer for Debian') {
                     when  {
                         beforeAgent true
                         anyOf {
-                            expression{ params.DISTRO == 'all' }  
-                            expression{ params.DISTRO == 'Debian' } // only trigger debian build
+                            expression { params.DISTRO == 'all' }
+                            expression { params.DISTRO == 'Debian' } // only trigger debian build
                         }
                     }
                     agent {
-                        label "${ARCH}" == "x86_64" ? "${env.NODE_LABEL_RPM}": "${env.NODE_LABEL_DEB}"
+                        label "${ARCH}" == 'x86_64' || "${ARCH}" == 'all' ? "${env.NODE_LABEL_RPM}" : "${env.NODE_LABEL_DEB}"
```

Comment:
Worth adding a comment why you're adding 'all' to this

Reply:
I realized this part is not really needed. after change the way how assign node for Deb arch, do not need to set agent in this step, it should use the default one from https://github.com/adoptium/installer/blob/cf227d2f7271ddd10555e5cbde5b09ec9b252591/linux/Jenkinsfile#L18